data_steward/retraction/retract_deactivated_pids.py

```diff
@@ -52,6 +33,37 @@ def get_client(project_id):
 WHERE person_id = {{pid}}
 """)
 
+# Queries to create tables in associated sandbox with rows that will be removed per cleaning rule
```

Comment:
These queries are not being used yet, these should belong in a PR where all the queries are being updated.

Reply:
I need them for the next ticket, like the util functions. I think they can be moved to a new PR for the main ticket DC-963. What are your thoughts?

Reply:
yes, moving this query to that PR will help, and also I think the query will be changed when you are working on it.